Maquetado/scss/tech.scss

```diff
@@ -0,0 +1,46 @@
+.tech {
+  background-color: $green;
+  border-radius: 5px;
+  margin: 1em;
+  padding: 3em;
+  position: relative;
+  width: 15%;
+  
+  &:before {
+    content: "";
+    border-radius: 5px;
+    bottom: 0;
+    box-shadow: -11px -11px white, -12px -12px $green, -12px -10px $green, -10px -12px $green;
+    left: 0;
+    padding: 2em;
+    position: absolute;
+    right: 0;
+    top: 0;
+    z-index: -1;
+  }
+  
+  .img-tech {
+    height: $tech-logo-size;
+    width: 100%;
+  }
+  
+  .title-tech{
+    display: block;
+    font-size: $text-big;
+    margin: 10px 0;
```

Comment:
it's preferred to add a margin bottom in the image instead, and a margin bottom here. that way it would be easier to read in the dom